backend/User/views.py

```diff
@@ -95,7 +145,7 @@ def post(self, request):
 			login(request, user)
 			# set additional session data if necessary
 			request.session['is_authenticated'] = True
-			return Response({"message": "User login successful"}, status=status.HTTP_202_ACCEPTED)
+			return Response(UserOutputSerializer(user).data, status=status.HTTP_202_ACCEPTED)
```

Comment:
I don't know how django does it, but couldn't we have a message nonetheless to give context about what what successful?